README.md

```diff
@@ -1,23 +1,61 @@
 
-Depends on [lager](https://github.com/pivotal-golang/lager) and [gorilla/mux](https://github.com/gorilla/mux).
+This is the home of the Cloud Foundry Service Broker for Google Cloud Platform. For a demo of installing and using the broker,
+see [here](https://www.youtube.com/watch?v=8nc4624K91A&list=PLIivdWyY5sqKJ48ycao632rEDuVbFm8yJ&index=3)
 
-Requires Go 1.7 and the associated buildpack.
+## Background
 
-## Examples
+### Service Brokers
 
-See the [examples](https://github.com/GoogleCloudPlatform/gcp-service-broker/tree/master/examples/) folder.
+This product is a [Cloud Foundry Service Broker](https://docs.cloudfoundry.org/services/overview.html). It adheres
+to [v2.8](https://docs.pivotal.io/pivotalcf/1-7/services/api.html) of the Service Broker API.
 
-## Prerequisites
+### Google Cloud Platform (GCP)
 
-### Set up a GCP Project
+GCP is a cloud service provider. In addition to VMs and networking, many other useful services are available. The ones
+available through this Service Broker are:
+
+* [PubSub](https://cloud.google.com/pubsub/)
+* [Cloud Storage](https://cloud.google.com/storage/)
+* [BigQuery](https://cloud.google.com/bigquery/)
+* [CloudSQL](https://cloud.google.com/sql/)
+* [ML APIs](https://cloud.google.com/ml/)
+* [Bigtable](https://cloud.google.com/bigtable/)
+* [Spanner](https://cloud.google.com/spanner/)
+* [Stackdriver Debugger](https://cloud.google.com/debugger/)
+* [Stackdriver Trace](https://cloud.google.com/trace/)
+
+
+## Installation
+
+Requires Go 1.8 and the associated buildpack.
+
+* [Installing as a Pivotal Ops Manager tile](#tile)
+* [Installing as a Cloud Foundry Application](#cf)
+    * [Set up a GCP Project](#project)
+    * [Enable APIs](#apis)
+    * [Create a root service account](#service-account)
+    * [Set up a backing database](#database)
+    * [Set required env vars](#required-env)
+    * [Optional env vars](#optional-env)
+    * [Push the service broker to CF and enable services](#push)
+    * [(Optional) Increase the default provision/bind timeout](#timeout)
+
+
+### Installing as a Pivotal Ops Manager tile <a name="tile"></a>
+
+Documentation for installing as a Pivotal Ops Manager tile is available [here](http://docs.pivotal.io/partners/gcp-sb/index.html)
+
+### Installing as a Cloud Foundry Application <a name="cf"></a>
+
+#### Set up a GCP Project <a name="project"></a>
 
 1. go to [Google Cloud Console](https://console.cloud.google.com) and sign up, walking through the setup wizard
 1. next to the Google Cloud Platform logo in the upper left-hand corner, click the dropdown and select "Create Project"
 1. give your project a name and click "Create"
 1. when the project is created (a notification will show in the upper right), refresh the page.
 
-### Enable APIS
+#### Enable APIs <a name="apis"></a>
 
 1. Navigate to **API Manager > Library**.
 1. Enable the <a href="https://console.cloud.google.com/apis/api/cloudresourcemanager.googleapis.com/overview">Google Cloud Resource Manager API</a>
```

Comment:
so that the table of contents can link to it. Is there  a way to do that in markdown? I looked for one but couldn't find.